src/main/java/com/azure/cosmos/kafka/connect/CosmosDBConfig.java

```diff
@@ -50,7 +53,15 @@ public class CosmosDBConfig extends AbstractConfig {
     private static final String COSMOS_CONTAINER_TOPIC_MAP_DISPLAY = "Topic-Container map";
     private static final String COSMOS_CONTAINER_TOPIC_MAP_DOC =
             "A comma delimited list of Kafka topics mapped to Cosmos containers.\n"
-                    + "For example: topic1#con1,topic2#con2.";    
+                    + "For example: topic1#con1,topic2#con2.";
+
+    public static final String COSMOS_SINK_BULK_ENABLED_CONF = "connect.cosmos.sink.bulk.enabled";
+    private static final String COSMOS_SINK_BULK_ENABLED_DOC = "Flag to indicate whether Cosmos DB bulk mode is enabled for Sink connector. By default it is true.";
+    private static final boolean DEFAULT_COSMOS_SINK_BULK_ENABLED = true;
+
+    public static final String COSMOS_SINK_MAX_RETRY_COUNT = "connect.cosmos.sink.maxRetryCount";
+    private static final String COSMOS_SINK_MAX_RETRY_COUNT_DOC = "Cosmos DB sink write max retry attempts on failure. By default, will retry on transient erros for up to 10 times.";
```

Comment:
```suggestion
    private static final String COSMOS_SINK_MAX_RETRY_COUNT_DOC = "Cosmos DB sink write max retry attempts on failure. By default, will retry on transient errors for up to 100000 times.";
```

Reply:
discussed offline, will keep the current value as it is for now, may change later